ONSdigital/eq-author-app
# Add or to checkbox routing chips

### What is the context of this PR?

Adds an `or` to the checkbox chip when adding options to the routing rules

### How to review 

1. Create questionnaire
2. Create a checkbox answer and add an `or`
3. Go to routing rules and add the or option
4. or should be displayed along with the option label
5. Remove the option label and see if it's replaced with `Unlabelled option`
